### DISCUSSION TITLE
Custom Markers

### DISCUSSION THREAD

**Discussion by muhammadsqln** [xclud/flutter_map#29] [Q&A] [answered]
Does this package allow for placing widgets as markers? 

Something like this: 

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/44453366/161647065-9ac79f12-65ca-465c-83f1-d51ea0d77215.png">

**Comment by xclud** [comment, marked as answer]
Yes it does. Just look and my markers example and use any other widget.
